packages/graphql-transformer-common/package.json

```diff
@@ -36,9 +36,9 @@
     "collectCoverage": true,
     "coverageThreshold": {
       "global": {
-        "branches": 30,
-        "functions": 19,
-        "lines": 30
+        "branches": 17,
```

Comment:
Per your convo https://github.com/aws-amplify/amplify-category-api/pull/1883#discussion_r1339237643 , are we committing to raising this before release? I'm pretty uncomfortable with the drop otherwise.

Reply:
The **effective** change in this package is a single line which I added tests for, not really sure what's causing this drop, but will add a task for this.